### PR TITLE
Update Cargo.toml

### PR DESCRIPTION
**Pull request by lukyjay**
Added slashes to the start of file locations to try address #277 

@nickbabcock I am new to this so I am unsure how to test if this fix will work or not. I am purely assuming that this is the solution to the issue I have raised. Please confirm :)

Thank you

### CODE REVIEW

**Comment by nickbabcock** [comment]
Thank you! Weird that you are receiving this error as I install with dpkg, but I've not yet tried it on 21.04. I'll make sure it works on 21.04 before I merge.

Can you see if you can dig up some debian packaging info that shows that these paths should be absolute paths (and what may have changed in 21.04 to cause this error). I bring this up as when packaging rpm files, one doesn't use absolute paths.

**Comment by lukyjay** [comment]
Hi

I found this: https://www.mail-archive.com/debian-dpkg-cvs@lists.debian.org/msg07446.html

I think new dpkg versions might require absolute paths and I assume dpkg was updated in 21.04? Just guessing.

21.04 seems to come with: 
```
root@transmission:~# dpkg --version 
Debian 'dpkg' package management program version 1.20.5 (amd64).
```

Just as a test - I have upgraded my laptop to ubuntu 21.04 (desktop) and it gives the same error

File: Cargo.toml [hunk 1]
```diff
@@ -54,9 +54,9 @@ features = ["vendored-openssl"]
...
-    ["assets/bare-config.toml", "etc/dness/dness.conf", "644"],
-    ["assets/dness.service", "etc/systemd/system/dness.service", "644"],
-    ["assets/dness.timer", "etc/systemd/system/dness.timer", "644"]
+    ["target/release/dness", "/usr/bin/dness", "755"],
+    ["assets/bare-config.toml", "/etc/dness/dness.conf", "644"],
+    ["assets/dness.service", "/etc/systemd/system/dness.service", "644"],
+    ["assets/dness.timer", "/etc/systemd/system/dness.timer", "644"]
 ]
-conf-files = ["etc/dness/dness.conf", "etc/dness/dness.env"]
+conf-files = ["etc/dness/dness.conf", "/etc/dness/dness.env"]
```
Comment (nickbabcock) [inline Cargo.toml:62]: I believe both of these conf-files paths should be absolute

Comment (nickbabcock) [reply Cargo.toml:62]: Based on the discussion here: https://github.com/mmstick/cargo-deb/issues/172 only the conf-files should be absolute paths, the other asset paths can be left as relative.

Comment (nickbabcock) [reply Cargo.toml:62]: Let me know if you need me to get this PR polished up.

Comment (lukyjay) [reply Cargo.toml:62]: Sorry not sure what needs to be done, appreciate if you can resolve :)

**Comment by nickbabcock** [comment]
Oops, accidentally closed this. I'm sorry. I opened up a new PR https://github.com/nickbabcock/dness/pull/286 with your commits and my finishing touches

**Pull request closed without merging**